plugins/auth-backend/package.json

```diff
@@ -89,6 +90,7 @@
     "@types/passport-saml": "^1.1.3",
     "@types/passport-strategy": "^0.2.35",
     "@types/xml2js": "^0.4.7",
+    "cross-fetch": "^3.1.5",
```

Comment:
Use `node-fetch` instead, since this is a backend package

Reply:
I tried to remove it, but somehow `msw` fails to mock my requests without [this line](https://github.com/backstage/backstage/pull/16186/files#diff-31f8175f56ceca064e3dd7177f22c8d1366d85cb01021d5d80d56864e5771816R30)

Reply:
nvm - I just figured it out :)